### ISSUE TITLE
Style-check rule for constants

### ISSUE THREAD

**Issue by lujiajing1126** [apache/skywalking#5362]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion
- [ ] Bug
- [ ] Requirement
- [x] Feature or performance improvement

___
### Requirement or improvement

As discussed in PR #5357 , any constant class members MUST only contain capital characters, numbers and `_`, except that it can be named as `logger`.

The related code is listed below,

https://github.com/apache/skywalking/blob/bcc27eb6cf9e9a5b01fbbd0a9899692185d3053d/apm-checkstyle/checkStyle.xml#L82

We reach an agreement that this additional rule should be removed and the code should be adapted to the new style. 

A new PR has to be created to focus on this issue. Also, this new PR should be merged before #5357 .
